File: get-set-full-stack/client/src/components/Track/Track.js
import React from "react";
import { useState, useEffect } from "react";
import { useNavigate } from "react-router-dom";
import './Track.css'

export const Track = () => {
    const [track, setTrack] = useState({
        id: 1,
        name: "Trying",
        bpm: 120,
        Notes: "don't rush",
        userProfileId: 1,
        BandId: 1,
        runTime: "00:03:20"
    })

    return (
        <div className="trackContainer">
            <div className="trackName">
                <span>{track.name}</span>
            </div>
            <div className="trackBpm">
                <span>{track.bpm} BPM</span>
            </div>
        </div>

    )
}

File: get-set-full-stack/Models/Track.cs
namespace get_set_full_stack.Models
{
    public class Track
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public int Bpm { get; set; }
        public int UserProfileId { get; set; }
        public UserProfile UserProfile { get; set; }
        public int BandId { get; set; }
        public string BandName { get; set; }
    }
}
